src/odinapi/views/level2.py

```diff
@@ -1234,3 +1260,27 @@ def parse_parameters(**kwargs):
         'areas': circles or area,
         'fields': fields
     }
+
+
+def get_level2view_paging_link(
+        url, param, product, next_min_scanid):
+    parameters_for_link = [
+        'min_lat', 'max_lat',
+        'min_lon', 'max_lon',
+        'min_pressure', 'max_pressure',
+        'min_altitude', 'max_altitude',
+        'start_time', 'end_time',
+        'min_scanid', 'document_limit'
+    ]
+    parameters = {}
+    for parameter in parameters_for_link:
+        if parameter in param and param[parameter] is not None:
+            parameters[parameter] = param[parameter]
+    parameters['product'] = product
+    parameters['min_scanid'] = 0
+    link = {}
+    link['first'] = make_rfc5988_link(url, **parameters)
```

Comment:
first and next are very strange link pairs.

Reply:
these are the simple ones, but for the intended usage next is the important one